packages/web/src/components/molecules/WithdrawCard/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import BigNumber from 'bignumber.js'
+import { Card, Button } from 'antd'
+
+interface Props {
+  label: 'Staking' | 'Holder'
+  amount: BigNumber
+  lastUpdate: string
+  onSubmitWithdraw: () => void
+}
+
+export const WithdrawCard = ({ amount, label, onSubmitWithdraw, lastUpdate }: Props) => {
+  return (
+    <Card
+      title={`Withdraw ${label} Reward`}
+      extra={
+        <>
+          <p>Last Update: ${lastUpdate}</p>
```

Comment:
Is the unit of measure for "Last Update" in `$` (`dollars`) ?

Reply:
oh this is my mistake. I will fix !